src/ProjectTemplates/test/SpaTemplateTest/AngularTemplateTest.cs

```diff
@@ -14,7 +15,15 @@ public AngularTemplateTest(ProjectFactoryFixture projectFactory, BrowserFixture
             : base(projectFactory, browserFixture, output) { }
 
         [Fact(Skip = "https://github.com/aspnet/AspNetCore-Internal/issues/1854")]
```

Comment:
If we think this improves the reliability of these tests we should un-skip all the spa tests. If we don't think the reliability is high enough yet the ones below should all be skipped too (if they're un-skipped to test the reliability then they should also all be un-skipped).

Reply:
I'm trying out stuff bro, it's the WIPs of the WIPs